db/migrate/20170719204710_create_pieces.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePieces < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pieces do |t|
+      t.integer :game_id
```

Comment:
@mscoutermarsh , can you please check this again and see if I added the index correctly? 

Thanks!